docs/.vuepress/containers/sourceCodeLink.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Matches into: `source-code-link https://github.com/handsontable/handsontable`.
+ *
+ * @type {RegExp}
+ */
+const exampleRegex = /^(source-code-link)\s*(\S*)\s*([\S|\s]*)$/;
+
+module.exports = {
+  type: 'source-code-link',
+  render(tokens, index) {
+    const token = tokens[index];
+
+    const [,, href] = token.info.trim().match(exampleRegex) || [];
+
+    if (href) {
+      // opening tag
+      return `<a href="${href}" target="_blank" class="source-code-link"> Source code <OutboundLink /> </a>`;
```

Comment:
It is the same way how vuperess adds this icon.

Reply:
Within vuepress theme or content? Because we have content here and this will make it less transferable to other markdown engines / readers / publishers

Reply:
This is a md container implementation which in assumptions works only for vuepress.

To be more transferable we have to additionally reject all Vuepress markdown extensions like `::: tip`, or our HelloWord component.  I don't think that is our main target which we focused.

Reply:
Vuepress extensions like `::: tip` are also available in other engines and are part of the content markup. We contain the markup to Markdown extensions. Here we go beyond that. I don't see how Vue component `<OutboundLink/>` is alike.

Reply:
It doesn't hold back the initial release. Let's do it at your earliest convenience @Siemienik

Reply:
REported in https://github.com/handsontable/handsontable/issues/8424#issue-942034118.